### DISCUSSION TITLE
osu!lazer should allow rebinding debug logs visible key (ctrl-F10)

### DISCUSSION THREAD

**Discussion by moandor-y** [ppy/osu#25793] [Ideas]
If a user uses the ctrl-F10 hotkey in another program (e.g. I use it for OBS split recording) while osu!lazer is the active window, it also makes the debug logs visible/hidden. This keyboard binding cannot be changed in key configurations.

**Comment by nagi-desuuu** [comment]
That's because the keybind is hardcoded into the framework the game is running on.

**Comment by peppy** [comment]
Eventually we might turn off these kinds of hotkeys for release builds.
